app/controllers/green_lanes/update_notifications_controller.rb

```diff
@@ -0,0 +1,25 @@
+module GreenLanes
+  class UpdateNotificationsController < AuthenticatedController
+    include XiOnly
+
+    before_action :disable_service_switching!
+    before_action :check_service
+    def index
+      @updates = GreenLanes::UpdateNotification.all(page: current_page).fetch
+    end
+
+    def edit
```

Comment:
This could probably be new action to be honest which pushes to an edit/update action.

Reply:
ignore me sorry read below a bit more